### PR TITLE
Sarah rami27/footer

### LINKED ISSUE #78: Responsive Footer
image

- Create a Responsive footer, with the following icons: Instagram, Discord, Slack, Youtube, Github
- Link the icons to their appropriate social media

### PR DESCRIPTION
**Pull request by Sarah-Rami27**
Here's what the footer looks like:
Screen Shot 2022-08-15 at 7 06 20 PM

Here's what it looks like when you hover over the icons:
Screen Shot 2022-08-15 at 7 07 26 PM
Screen Shot 2022-08-15 at 7 07 13 PM
Screen Shot 2022-08-15 at 7 07 06 PM
Screen Shot 2022-08-15 at 7 07 01 PM

If the hover colors need to be changed I can fix them.
Closes #78

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,7 +1,81 @@
...
+				</h2>
+				<div className='flex flex-wrap justify-center justify-around px-80'>
+					<a
+						rel='noopener noreferrer'
+						href='https://www.instagram.com/acm_ucr/'
+						target='_blank'
+						title='Instagram'
+						className='w-12 h-12 mx-10 my-6 fill-current md:mx-12 lg:m-0 dark:text-gray-50 hover:text-purple-500'
+					>
+						<svg
```
Comment (shahdivyank) [inline components/Footer.js]: What is the SVG for? Can we switch over to using React Icons? Same goes for the path component

File: components/Footer.js [hunk 2]
```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
+			<div className='container p-6 mx-auto lg:space-y-8'>
+				<h2 className='text-4xl font-lexend font-bold text-center'>
```
Comment (shahdivyank) [inline components/Footer.js]: Switch out the `<h2>` for a `<p>` tag

File: components/Footer.js [hunk 3]
```diff
@@ -1,7 +1,81 @@
...
+					Let&#39;s connect!
+				</h2>
+				<div className='flex flex-wrap justify-center justify-around px-80'>
```
Comment (shahdivyank) [inline components/Footer.js]: What is the `justify-around` and `px-80` for? `px-80` is a dangerously large margin value and breaks on mobile, where the icons aren't centered: 
image

File: components/Footer.js [hunk 4]
```diff
@@ -1,7 +1,81 @@
...
+					<a
+						rel='noopener noreferrer'
+						href='https://www.instagram.com/acm_ucr/'
+						target='_blank'
+						title='Instagram'
+						className='w-12 h-12 mx-10 my-6 fill-current md:mx-12 lg:m-0 dark:text-gray-50 hover:text-purple-500'
```
Comment (shahdivyank) [inline components/Footer.js]: `w-12` and `h-12` are also dangerous values to be working with and the margin levels as well. Use percentages when possible so its scales nicely, same applies for margins. 

Margins are not enough (to replicate, just try resizing your window): 
image

Breaks into Column Very Quickly:
image

**Review by shahdivyank — CHANGES_REQUESTED** [review]
I know the design doesn't include it initially, but can you also add a SLACK, MEDIUM social media icons as well! To tackle the responsiveness problem, I would recommend implementing a Bootstrap Grid system, where each Col is a social media icon. As per width, you can leave it to be a percentage of the entire screen and left Grid handle the margins

File: components/Footer.js [hunk 5]
```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
```
Comment (shahdivyank) [inline components/Footer.js]: No need to put the `dark:` that is for dark/light themes which is not something we will be doing currently, just `bg-neutral-900` is good!

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Comment by Sarah-Rami27** [comment]
What it looks like with the added icons:
Screen Shot 2022-08-16 at 5 19 38 PM

What it looks like with the screen smaller:
Screen Shot 2022-08-16 at 5 19 55 PM

File: components/Footer.js [hunk 6]
```diff
@@ -1,7 +1,95 @@
...
+		<footer className='bg-neutral-900 text-gray-50 py-6'>
+			<div className='grid sm:grid-rows-2 justify-center mx-auto'>
+				<div>
+					<p className='text-4xl font-lexend font-bold text-center'>
+						Let&#39;s connect!
+					</p>
+				</div>
+				<div>
+					<div className='grid sm:grid-cols-6 grid-cols-3 justify-items-center mx-auto gap-4'>
+						<div>
```
Comment (shahdivyank) [inline components/Footer.js]: What are these div's for? The ones without a class name, you can remove them and just have the a tag. Less code -> lighter website

File: components/Footer.js [hunk 7]
```diff
@@ -1,7 +1,95 @@
...
+	FaYoutube,
+	FaMediumM,
+	FaGithub,
+} from "react-icons/fa";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='bg-neutral-900 text-gray-50 py-6'>
+			<div className='grid sm:grid-rows-2 justify-center mx-auto'>
```
Comment (shahdivyank) [inline components/Footer.js]: Use React Bootstrap Grid with Row and Col components

**Review by shahdivyank — CHANGES_REQUESTED** [review]

File: components/Footer.js (hunk 7 above)
Comment (shahdivyank) [reply components/Footer.js]: https://github.com/acm-ucr/acm-dev/blob/pondohoo/filter/components/Calendar.js 

 Check this example out!

**Review by shahdivyank — APPROVED** [review]

**Review by SamuelLin213 — APPROVED** [review]

**Pull request merged**